### DISCUSSION TITLE
Potential Bug: marp-vscode - exporting to Powerpoint disables external links

### DISCUSSION THREAD

**Discussion by jtcmedia** [marp-team/marp#166] [Q&A] [answered]
When I have a slide using standard markdown for external links, it shows up as the light-blue link color in the exported Powerpoint but it not clickable. Exporting to html works fine.

**Comment by yhatt** [comment, marked as answer]
_It's not a bug, just a technical limitation._ Marp's PPTX export has consisted with pre-rendered slides as background images, just as same as [GitPitch](https://medium.com/@gitpitch/gitpitch-embraces-microsoft-powerpoint-keynote-google-slides-7095017dd57c).  These images have no interactive informations like a hyperlink so you can never interact.

This limitation should be no problems for typical presenting because Marp has designed to output "static slides", but you should use PDF format if you want some hyperlinks. We treat PPTX export as just an auxiliary format. I'd like you to think it's the result after doing the best for PPTX generation.

If you really need to use hyperlinks in PPTX, we always recommend to convert exported PDF to PPTX. See #82 in "Frequently Asked" section.

### Related

- #82
- https://github.com/marp-team/marp-cli/issues/166
- https://github.com/marp-team/marp-cli/issues/196
- https://github.com/marp-team/marp-cli/issues/249
- https://github.com/marp-team/marp-cli/issues/298
- https://github.com/marp-team/marp-vscode/issues/139

**Reply by jtcmedia** [reply]
Thanks for the explanation.
